### ISSUE TITLE
[data] ensure correct `output` from #71

### ISSUE THREAD

**Issue by jGaboardi** [uscuni/simplification#76]
[data] ensure correct `output` from #71

* xref https://github.com/uscuni/simplification/issues/70#issuecomment-2186461796

**Comment by dancejod** [comment]
> So here's the layers that are available for our FUA's:
> 
> ```
> Aleppo
>   - ['roads_output', 'polygons_output']
> Auckland
>   - ['roads_input', 'polygons_output', 'roads_output']
> Bucaramanga
>   - ['roads_input', 'main']
> Douala
>   - ['roads_input', 'roads_output']
> Liège
>   - ['roads_output', 'roads_input']
> Salt Lake City
>   - ['roads_output', 'polygons', 'roads_input']
> Wuhan
>   xx gpkg_in.exists()=False
> ```
> 
> We have a `roads_output` for all FUA accept _Bucaramanga_ and also _Wuhan_ (because it doesn't exist yet)
> 
> @Kryndlea @dancejod Can either of you confirm that `main` in _Bucaramanga_ is simply a renamed version of `roads_output`?

@jGaboardi that's right, it is supposed to be `roads_output`, double checked just in case👍

**Comment by jGaboardi** [comment]
OK, cool.
